BrushUpCode/src/Coffee.java

```diff
@@ -0,0 +1,272 @@
+                System.out.println("PLease enter an instruction: take, fill, buy, remaining, exit");
+                input = enter.nextLine();
+
+            } else if (input.equalsIgnoreCase(remaining)) {
+                remaining(dollar, water, milk, coffeeBeans, cups);
+                System.out.println("PLease enter an instruction: take, fill, buy, remaining, exit");
+                input = enter.nextLine();
+            }
+        }
+        System.out.println("thank you for using our services!");
+        this.coffeeMachineState = State.OFF;
+        System.exit(0);
+    }
+
+
+    /*method for instruction take*/
+    private void take(int dollar) {
+        System.out.println("I gave you " + dollar + "$");
+        dollar -= dollar;
+        setDollar(dollar);
+
+    }
+
+    /*method for instruction fill*/
+    private void fill(int water, int milk, int coffeeBeans, int cups) {
+        this.coffeeMachineState = State.FILLING;
+        Scanner enter = new Scanner(System.in);
+        System.out.println("Write how many ml of water do you want to add:");
+        int addedWater = enter.nextInt();
+        water += addedWater;
+        setWater(water);
+        System.out.println("Write how many ml of milk do you want to add:");
+        int addedMilk = enter.nextInt();
+        milk += addedMilk;
+        setMilk(milk);
+        System.out.println("Write how many grams of coffee beans do you want to add");
+        int addedCoffee = enter.nextInt();
+        coffeeBeans += addedCoffee;
+        setCoffeeBeans(coffeeBeans);
+        System.out.println("Write how many disposable cups of coffee do you want to add: ");
+        int addedCups = enter.nextInt();
+        cups += addedCups;
+        setCups(cups);
+
+    }
+
+    /*method for instruction buy, consists of three different methods to buy espresso latte and cappuccino*/
+    private void buy(int dollar, int water, int milk, int coffeeBeans, int cups) {
+        Scanner enter = new Scanner(System.in);
+        System.out.println("What do you want to buy? 1) espresso 2) latte 3) cappuccino ? If mistaken you can go back by instruction  4) back");
+        int choice = enter.nextInt();
+        this.coffeeMachineState = State.MAKINGCOFFE;
+        switch (choice) {
+            case 1: {
+                System.out.println("Good Choice! ");
+                buyEspresso(dollar, water, milk, coffeeBeans, cups);
+                break;
+            }
+
+            case 2: {
+                System.out.println("Yummy Latte!");
+
+                buyLatte(dollar, water, milk, coffeeBeans, cups);
+                break;
+            }
+
+            case 3: {
+                System.out.println("Great choice!");
+                buyCappuccino(dollar, water, milk, coffeeBeans, cups);
+                break;
+            }
+            case 4: {
+                break;
+            }
+        }
+    }
+
+
+    /*method for buying espresso(used in method buy)*/
```

Comment:
@maria-salmasian jan you can use java doc comments which will add parameters as well just simply write `/**` press enter and it will be generated and then in the comment you can add your comment as well

Reply:
done!